Easydict/NewApp/LanguagePreference/EZI18nHelper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  EZI18nHelper.swift
+//  Easydict
+//
+//  Created by choykarl on 2024/3/4.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import SwiftUI
+
+// MARK: - EZI18nHelper
+
+@objcMembers
+class EZI18nHelper: NSObject {
+    static let languagePreferenceChangedNotification = "EZLanguagePreferenceChangedNotification"
```

Comment:
It would be better to define this notification name in `Notification+Name.swift`.